modules/ROOT/pages/salesforce/salesforce-connector-reference-97.adoc

```diff
@@ -2220,27 +2220,29 @@ Retrieves the list of individual records that have been created or updated withi
 
 [[getUpdatedObjects]]
 === Get Updated Objects
-`<salesforce:get-updated-objects>`
 
+`<salesforce:get-updated-objects>`
 
-Retrieves the list of records that have been updated between the last time this method was called and now. This method will save the timestamp of the latest date covered by Salesforce represented by `GetUpdatedResult#latestDateCovered`. 
+Retrieves a list of records that have been updated between the last time this method was called and the current server's timestamp. 
 
-IMPORTANT: In order to use this method in a reliable way user must ensure that right after this method returns the result is stored in a persistent way since the timestamp of the latest. To reset the latest update time use `resetUpdatedObjectsTimestamp(String)`.
+The Salesforce API ignores the seconds portion of dates, so subsequent calls of this operation should take place at a rate of at least one minute, otherwise the operation does not return any results for the subsequent calls.
```

Comment:
I suggest starting a new sentence with "Otherwise,..." The current sentence is long and hard to follow. If you don't want to do that, add a comma after "otherwise." 

Do you mean one call per minute? If so, "calls of this operation should take place at a rate of one per minute.